pkg/container/container_test.go

```diff
@@ -213,6 +213,28 @@ var _ = Describe("the container", func() {
 					Expect(links).To(SatisfyAll(ContainElement("redis"), ContainElement("postgres"), HaveLen(2)))
 				})
 			})
+			When("there is a pre or post update timeout", func() {
+				It("should return minute values", func() {
```

Comment:
```
				It("should return minute values", func() {
```

copy/paste artifact?

Reply:
it applies to one of the env variables, but it should probably be split into 
```suggestion
				It("should return the minute value", func() {
```
and 
```go
			When("there is a pre or post update user", func() {
				It("should return the specified user", func() {
```